packages/react-native/ReactCommon/react/renderer/components/textinput/platform/ios/react/renderer/components/iostextinput/TextInputEventEmitter.h

```diff
@@ -44,6 +44,23 @@ class TextInputEventEmitter : public ViewEventEmitter {
   void onKeyPress(const KeyPressMetrics& keyPressMetrics) const;
   void onScroll(const Metrics& textInputMetrics) const;
 
+#if TARGET_OS_OSX // [macOS
+  struct OnAutoCorrectChange {
+      bool enabled;
+    };
+  void onAutoCorrectChange(OnAutoCorrectChange value) const;
+
+  struct OnSpellCheckChange {
+      bool enabled;
+    };
+  void onSpellCheckChange(OnSpellCheckChange value) const;
+
+  struct OnGrammarCheckChange {
+      bool enabled;
+    };
+  void onGrammarCheckChange(OnGrammarCheckChange value) const;
```

Comment:
Nit: Indentation

Reply:
What's wrong with it? It's two spaces in like the stuff outside the block

Reply:
Nvm, I see it now